src/Tasks/TaskTranslatorHelpers.cs

```diff
@@ -34,5 +35,35 @@ public static void Translate(this ITranslator translator, ref FrameworkName fram
                 frameworkName = new FrameworkName(identifier, version, profile);
             }
         }
+
+        public static void TranslateDictionary(this ITranslator translator, ref Dictionary<string, DateTime> dict, StringComparer comparer)
+        {
+            int count = 0;
+            if (translator.Mode == TranslationDirection.ReadFromStream)
+            {
+                dict = new Dictionary<string, DateTime>(comparer);
+                translator.Translate(ref count);
+                string key = string.Empty;
+                DateTime val = DateTime.Now;
```

Comment:
Would it be harmful to default this to min or max? Those are constants and would never require a clock read.

```suggestion
                DateTime val = DateTime.MinValue;
```

Reply:
Nope, just needed a value.